reinforcement_learning/rlclientlib/live_model_impl.cc

```diff
@@ -177,9 +177,9 @@ namespace reinforcement_learning {
     // Generate egreedy pdf
     size_t action_count = 0;
     RETURN_IF_FAIL(utility::get_action_count(action_count, context, _trace_logger.get(), status));
-    
+
     vector<float> pdf(action_count);
-    // Generate a pdf with epsilon distributed between all action.  The top action 
+    // Generate a pdf with epsilon distributed between all action.  The top action
```

Comment:
[ultra minor] would be easier to read if ` The top action` is moved to the next line